serverless.js

```diff
@@ -101,7 +127,14 @@ class Website extends Component {
 
     this.state.bucketName = inputs.bucketName
     this.state.region = inputs.region
-    this.state.url = `http://${bucketOutputs.name}.s3-website-${inputs.region}.amazonaws.com`
+    
+    const endpointPattern = (
+      endpointsPatterns[inputs.region] 
+      || endpointsPatterns[null]
+    ).replace('%region%', inputs.region);
```

Comment:
can't we do this with a `getUrl` function and pass it a region? We then use string literals. It's more readable this way imo.

Reply:
More readable, you're right